src/motioncore/utility/meta.hpp

```diff
@@ -64,4 +64,31 @@ using TypeMap = decltype(detail::MakeTypeMap<Value, Ts...>());
 
 #endif
 
+// return the int type of uint type
+template <typename T>
```

Comment:
This can be replaced with `std::make_signed`

Reply:
Commit is reverted and further implementations use std::make_signed instead.